README.md

```diff
@@ -0,0 +1,35 @@
+# Awesome-books
+
+> A single page project with the porpuse of storing books' titles and authors.
+
+
+## Built With
+
+- CSS, HTML & Javascript. 
+
+
+## Authors
+
+👤 **Saadat Ali**
+
+- GitHub: [@Saadat123456](https://github.com/Saadat123456)
+- Twitter: [@Saadat02021999](https://twitter.com/Saadat02021999)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/saadatali1999/)
+
+👤 **Alejandro Neira**
+
+- GitHub: [@AlejandroNeira98](https://github.com/AlejandroNeira98)
+- Twitter: [@JANP_19](https://twitter.com/JANP_19)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-neira-0b45b6226/)
+
+## 🤝 Contributing
+
```

Comment:
- [ ] I would strongly recommend you a section in your README file that gives instructions on how people can get and run your project locally.